src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![doc = include_str!("../README.md")]
 #![deny(missing_debug_implementations)]
 #![deny(unsafe_code)]
+#![deny(missing_docs)]
```

Comment:
noooooooooooo. I can't be lazy anymore :(

Reply:
You know it crossed my mind that working on the code may be annoying locally since every pub item needs a comment now to build :thinking: 

Maybe we can require that only in CI or provide some `lazy` feature flag? :wink:

Reply:
yeah I usually comment it out in dev.

Reply:
```suggestion
#![cfg_attr(doc, deny(missing_docs))]
```

Maybe something like this would work. It'd enable missing docs lint only when documenting. And documenting is part of the CI script.

(I didn't test that, I'm on mobile now 😅)

Reply:
I don't think we should bother. This is alright as it is.